### PR TITLE
useCopyClipboard - introduce browser api polyfills to cover CopyClipboard in JSDOM

### PR DESCRIPTION
**Pull request by mykas**
Component CopyClipboard is rendered by our consumers in JSDOM environment. We should provide JSDOM based polyfills.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6628

**Comment by shlomitc** [comment]
Following @carmelc comments - even after adding the polyfills (which are good), he will still not be able to interact with the component in unit tests. So all of these are just not to fail the test, but need to keep that in mind

**Comment by mykas** [comment]
Partially he will be able to do that, because CopyClipboard does include a div that can be run against with methods like: `exists`.

File: packages/wix-style-react/testkit/polyfills/selection-behavior.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const install = () => {
     return {
       addRange: () => {},
       removeAllRanges: () => {},
+      selectNodeContents: () => {},
```
Comment (egidijusja) [inline packages/wix-style-react/testkit/polyfills/selection-behavior.js:13]: Do we really need this? I checked w3c specs and couldn't find that getSelection factory would have selectNodeContents function.

**Review by egidijusja — APPROVED** [review]
Looks good to me!

**Pull request merged**